### PR TITLE
python3-gpsd-py3: Added recipe

### PR DESCRIPTION
**Pull request by colin-pm**
Create Python3 recipe for gpsd-py3.

Signed-off-by: Colin McAllister <colinmca242@gmail.com>

### CODE REVIEW

File: meta-python/recipes-devtools/python/python3-gpsd-py3_0.3.0.bb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+SRC_URI[sha256sum] = "2908d3bd78dfb6720ecfe22f97e139b5a4a198f38df3a77215cf644a33513192"
+
+PYPI_PACKAGE = "gpsd-py3"
+
+inherit pypi setuptools3
+
+CLEANBROKEN = "1"
+
+RDEPENDS_${PN} += " \
+    gpsd \
```
Comment (kraj) [inline meta-python/recipes-devtools/python/python3-gpsd-py3_0.3.0.bb:15]: gpsd comes from meta-oe layer, but its not present in layer dependencies. for meta-python, so this should be added in layer.conf

Comment (kraj) [reply meta-python/recipes-devtools/python/python3-gpsd-py3_0.3.0.bb:15]: Also add it to packagegroup-meta-python since its a new recipe

**Review by kraj — CHANGES_REQUESTED** [review]

File: meta-python/recipes-devtools/python/python3-gpsd-py3_0.3.0.bb (hunk 1 above)
Comment (colin-pm) [reply meta-python/recipes-devtools/python/python3-gpsd-py3_0.3.0.bb:15]: On second thought, I don't think gpsd is a runtime dependency.  I forgot that you can connect to a remote gpsd service, so I will remove gpsd from the runtime dependencies.

I'll add the package to packagegroup-meta-python.  To confirm, that's in meta-python/recipes-core/packagegroups/packagegroup-meta-python.bb, correct?

Comment (kraj) [reply meta-python/recipes-devtools/python/python3-gpsd-py3_0.3.0.bb:15]: right thats the packagegroup recipe to add to

**Comment by colin-pm** [comment]
gpsd has been removed from runtime dependencies and I added python3-gpsd-py3 to packagegroup-meta-python

**Comment by kraj** [comment]
this should be fixed now with 9991671eb3

**Pull request closed without merging**